.github/workflows/wheels-test.yml

```diff
@@ -152,11 +168,48 @@ jobs:
         date: ${{ inputs.date }}
         sha: ${{ inputs.sha }}
 
-    - name: Set CTK-related vars from input CTK versions
-      uses: rapidsai/shared-action-workflows/wheel-ctk-name-gen@branch-23.06
-      with:
-        ctk: ${{ matrix.ctk }}
-        package-name: ${{ inputs.package-name }}
-
     - name: Run citestwheel
-      run: /citestwheel.sh
+      run: |
+        set -eoxu pipefail
+
+        ctk_tag="${{ matrix.ctk }}"
+        ctk_major=$(echo $ctk_tag | cut -d'.' -f1)
+        cuda_suffix="cu${ctk_major}"
```

Comment:
This can use [the new gha-tool that](https://github.com/rapidsai/gha-tools/pull/55) @divyegala added. [Here's how it's being used now](https://github.com/rapidsai/shared-action-workflows/pull/91/files).